src/Mutation.jl

```diff
@@ -1,7 +1,58 @@
 module Mutation
 
 using ..Types
+using Distributions
 
-# TODO
+"""
+    Defines the mutation of genes of real values (currently only Float64 and Integer types (including Bool))
+    Mutation is applied for each gene on its own with probability `mutation_rate` vith values from the uniform
+    distribution in the interval 'mutation_interval'.
+"""
+struct RealGeneMutation{T<:Union{Float64, Integer}}
+    mutation_rate::Float64
+    mutation_interval::Tuple{T, T}
 
+    function RealGeneMutation(mutation_rate::Float64, mutation_interval::Tuple{T, T}) where T<:Real
```

Comment:
Why do you have lower type restrictions on the constructor (Real) than on the struct itself  (only Float64 and Integer)?